### ISSUE TITLE
No namespace: cljs.core found

### ISSUE THREAD

**Issue by soulawaker** [philoskim/debux#6]
I tried following things in an plain vanilla project for verification. Have I missed anything?

**Try and exceptions**

```clojure
$ lein repl
user=> (use 'debux.core)
CompilerException java.lang.Exception: No namespace: cljs.core found, compiling:(debux/cs/macro_types.cljc:5:1)
```
\
**Environments and dependencies**
Leiningen: 2.8.1
JVM: 1.8, 64bit 
```clojure
:dependencies [[org.clojure/clojure "1.8.0"]
               [org.clojure/clojurescript "1.9.854"]
               [philoskim/debux "0.4.6"]]
```

**Comment by philoskim** [comment]
I fixed the exception error and released the newer version (0.4.7).

Thanks for the reporting!

**Comment by soulawaker** [comment]
Thanks. It worked! I close this issue.
